go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/zmap/zlint
+
+require (
+	github.com/asaskevich/govalidator v0.0.0-20180720115003-f9ffefc3facf
```

Comment:
It looks like this dependency is only present because of `govalidator.IsURL` being used in `util/fqdn.go`. Similarly `zcrypto` carries a `govalidator` dep just for `IsURL`.

It might be nice to knock off a dependency off of both projects by implementing this one function in `zcrypto`.

Reply:
I opened https://github.com/zmap/zcrypto/issues/187 on `zcrypto` for this.